p2p/src/test/java/com/google/android/fhir/p2p/SyncPayloadTest.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.fhir.p2p
+
+import ca.uhn.fhir.context.FhirContext
+import com.google.common.truth.Truth.assertThat
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import org.hl7.fhir.r4.model.Patient
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.BlockJUnit4ClassRunner
+
+@RunWith(BlockJUnit4ClassRunner::class)
+class SyncPayloadTest {
+  @Test
+  fun encodeToStringEncodesThePayloadToAJsonString() {
```

Comment:
Do you not think that it is worth adding a decode test too?

Reply:
Ah yes, that's the other test I wanted to add. Will do it now quickly.

Reply:
I've added a test for decoding the payloads in https://github.com/decode-development/android-fhir/pull/1/commits/3e4804e45deaca20ef623525e3de2fcd16fbc27d.